src/main/resources/public/ts/services/minetest.service.ts

```diff
@@ -45,15 +45,29 @@ export const minetestService: IMinetestService = {
         return http.put(`/minetest/worlds`, worldBody);
     },
 
-    delete: (worlds: Array<String>): Promise<AxiosResponse> => {
-        let idParams: string = '';
-        if (worlds.length > 0) {
-            worlds.forEach((_id: string) => {
-                idParams += `&id=${_id}`;
-            });
-        }
+    //TODO keep logic for later (multidelete)
+    // delete: (worlds: Array<String>): Promise<AxiosResponse> => {
+    //     let idParams: string = '';
+    //     if (worlds.length > 0) {
+    //         worlds.forEach((_id: string) => {
+    //             idParams += `&id=${_id}`;
+    //         });
+    //     }
+    //
+    //     return http.delete(`/minetest/worlds?${idParams}`);
+    // }
+
+    delete: (world: IWorld): Promise<AxiosResponse> => {
+
+        //TODO keep logic for later (multidelete)
```

Comment:
commentaire à enlever ?

Reply:
Multidelete removed for now but has to be used later, so we keep logic comment here